media-gfx/displaycal/metadata.xml

```diff
@@ -1,11 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="person">
-    <email>gentoo@xayto.net</email>
-    <name>Craig Inches</name>
-    <description>Proxied maintainer; set to assignee in all bugs</description>
-  </maintainer>
```

Comment:
Can you remove proxy-maint here as well and insert `<!-- maintainer-needed -->`?